types/+equMPC/code_equMPC_ADMM_C.c

```diff
@@ -112,18 +115,24 @@ void equMPC_ADMM(double *pointer_x0, double *pointer_xr, double *pointer_ur, dou
     for(unsigned int i = 0; i < nn; i++){
         x0[i] = scaling_x[i]*( pointer_x0[i] - OpPoint_x[i] );
         xr[i] = scaling_x[i]*( pointer_xr[i] - OpPoint_x[i] );
+        // TODO: Get LB and UB when in_engineering == 1
     }
     for(unsigned int i = 0; i < mm_; i++){
         ur[i] = scaling_u[i]*( pointer_ur[i] - OpPoint_u[i] );
+        // TODO: Get LB and UB when in_engineering == 1
     }
     #endif
     #if in_engineering == 0
     for(unsigned int i = 0; i < nn; i++){
         x0[i] = pointer_x0[i];
         xr[i] = pointer_xr[i];
+        LB[i] = pointer_LB[i];
```

Comment:
Lines `129-130` can be merged with lines `134-135` into a single `for` loop. The only reason to have them separate would be if we had `LBx`, `UBx`, `LBu`, `UBu` as separate inputs in the C function.